warehouse/warehouse.go

```diff
@@ -18,3 +20,18 @@ type Warehouse interface {
 	GetUploadFailedMsg(filename string, err error) string
 	IsUploadOnly() bool
 }
+
+// Common implementation that subclasses can call
+func ValueToString(val interface{}, isTime bool) string {
```

Comment:
So right now each implementation 😃 just calls the method from `warehouse` (except for redshift, that looks sufficiently different). Is there a better way of doing this? I just hate how otherwise we'd have the same exact piece of code replicated in multiple implementations.

Reply:
You've done it the way that it should be done; let me be more clear on the changes I was suggesting:

1. Change the comment to "Common logic that implementations can call" (i.e. remove the word "subclasses" from the comment.
2. Change the name of the function to `valueToString` so that it is not exported from the package, and thus is only callable via one of the exported implementations that calls it.

Reply:
Ah, makes sense! Will fix.